pkg/controller/che/finalizer.go

```diff
@@ -27,7 +27,7 @@ func (r *ReconcileChe) ReconcileFinalizer(instance *orgv1.CheCluster) (err error
 			if err := r.client.Delete(context.TODO(), oAuthClient); err != nil {
 				logrus.Errorf("Failed to delete %s oAuthClient: %s", oAuthClientName, err)
 				return err
-			}
+			}		
```

Comment:
I think this extra witespaces are not needed

Reply:
fixed